app/models/course.rb

```diff
@@ -260,6 +260,27 @@ def scholarship_amount
     dfe_subjects&.first&.scholarship_amount
   end
 
+  def editable_enrichment
```

Comment:
Not sure this is the right place to be putting this logic.

@misaka would this be a good candidate for an association method?

Reply:
Good call